### DISCUSSION TITLE
Clarity needed on filter usage

### DISCUSSION THREAD

**Discussion by causeway74** [staxrip/staxrip#1156] [General]
Hi, I can easily go into the "edit code" option to see the list of filters that are selected for an operation. 

If I were to include something like TemporalDegrain2 will the tool automatically handle any required conversion e.g. convertToYV16() which is a pre-req for using this filter?

Does this conversion happen automatically when I reference TD2 or do I need to add it explicitly? If yes, can I see the resulting script anywhere? I don't see it when selecting "Preview Code" and I don't get any errors running my script. 

Any guidance please?

**Comment by Dendraspis** [comment]
Color Depth conversions are not made automatically, you have to  add them manually in case it has to be done.

The "Preview Code" shows the resulting and complete generated script. So if there is no error apprearing and the preview is also ok, it looks like nothing has to be added/changed.

**Reply by causeway74** [reply]
Helpful, thank you.
